src/pages/workspaces/workspace/notebooks/NotebookLauncher.js

```diff
@@ -445,14 +445,14 @@ const NotebookEditorFrame = ({ mode, notebookName, workspace: { workspace: { nam
       h(JupyterFrameManager, {
         frameRef,
         onClose: () => Nav.goToPath('workspace-notebooks', { namespace, name }),
-        details: { notebookName, name, namespace }
+        details: { notebookName, name, namespace, googleProject }
```

Comment:
Looking again, it seems like `JupyterFrameManager` doesn't actually use `googleProject` so maybe it doesn't need to be passed here?

Reply:
Good catch! I wonder if we _should_ capture `googleProject` as well now that it may be distinct from `workspace.namespace`  🤔